sqsworkers/crew.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, **kwargs):
         self.worker_limit = kwargs['worker_limit'] if 'worker_limit' in kwargs else 10
         self.max_number_of_messages = kwargs['max_number_of_messages'] if 'max_number_of_messages' in kwargs else 1
         self.wait_time = kwargs['wait_time'] if 'wait_time' in kwargs else 20
+        self.bulk_mode = kwargs['bulk_mode'] if 'bulk_mode' in kwargs else False
```

Comment:
A comment explaining what exactly the new parameter does would be helpful to future users

Reply:
Also, add `bulk_mode` to the docs on line 49

Reply:
Bonus: add the docs for `self.exception_handler_function` (riddhi :stare:)